packages/appstore/src/components/ready-to-update-wallets/upgrade-info-list.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { TReadyToUpgradeForm } from './ready-to-upgrade-form';
+import { isMobile } from '@deriv/shared';
+
+type TUpgradeInformationList = {
+    name: string;
+    visiblity: boolean;
+    content: React.ReactElement;
+};
+
+const UpgradeInformationList = ({ is_high_risk, is_eu }: TReadyToUpgradeForm) => {
+    const list: TUpgradeInformationList[] = [
+        {
+            name: 'upgrade_info',
+            visiblity: true,
+            content: (
+                <Localize i18n_default_text='During the upgrade, deposits, withdrawals, transfers, and adding new accounts will be unavailable.' />
+            ),
+        },
+        {
+            name: 'open_positions',
+            visiblity: true,
+            content: (
+                <Localize i18n_default_text="Your open positions won't be affected and you can continue trading." />
+            ),
+        },
+        {
+            name: 'deriv_p2p',
+            visiblity: !is_eu,
+            content: (
+                <Localize
+                    i18n_default_text='<0>Deriv P2P</0> is not supported in wallets yet. This option will be unavailable until further notice.'
+                    components={
+                        <Text
+                            weight='bold'
+                            size={isMobile() ? 'xxs' : 'xs'}
+                            line_height={isMobile() ? 'm' : 'l'}
+                            key={0}
+                        />
+                    }
+                />
+            ),
+        },
+        {
+            name: 'regular_pa',
+            visiblity: !is_high_risk && !is_eu,
+            content: (
+                <Localize
+                    i18n_default_text="You can use <0>payment agents'</0> services to make deposits by adding a Payment agent wallet after the upgrade."
+                    components={
+                        <Text
+                            weight='bold'
+                            size={isMobile() ? 'xxs' : 'xs'}
+                            line_height={isMobile() ? 'm' : 'l'}
+                            key={0}
+                        />
+                    }
+                />
+            ),
+        },
+        {
+            name: 'bold_pa',
```

Comment:
why we need regular and bold? if it fallows the desgin, the bold part maybe missed there. please remove the regular one